nimbus-core/src/main/java/com/antheminc/oss/nimbus/integration/mq/MqMessage.java

```diff
@@ -0,0 +1,33 @@
+/**
+ *  Copyright 2016-2019 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.antheminc.oss.nimbus.integration.mq;
+
+import java.io.Serializable;
+
+import lombok.Data;
+
+/**
+ * @author Sandeep Mantha
+ * @author Tony Lopez
+ */
+@Data
+public class MqMessage implements Serializable {
```

Comment:
Could you look at AbstractEvent and see if the same can be used instead of creating another Message structure.

Reply:
Fixed.